govc/test/vm.bats

```diff
@@ -960,3 +960,39 @@ load test_helper
   run govc object.collect -s vm/DC0_H0_VM0 guest.ipAddress
   assert_success 10.0.0.45
 }
+
+@test "guest fileops" {
+  if ! docker version ; then
+    skip "docker client not installed"
+  fi
+
+  vcsim_env -autostart=false
+
+  export GOVC_VM=DC0_H0_VM0
+  name="vcsim-$GOVC_VM-$(govc object.collect -s "vm/$GOVC_VM" config.uuid)"
+
+  if docker inspect "$name" ; then
+    flunk "$GOVC_VM container still exists"
+  fi
+
+  run govc vm.change -e RUN.container=nginx
+  assert_success
+
+  run govc vm.power -on $GOVC_VM
+  assert_success
+
+  if ! docker inspect "$name" ; then
+    flunk "$GOVC_VM container does not exist"
+  fi
+
+  run govc guest.upload README.md /tmp/README.md
+  assert_failure # unauthenticated
```

Comment:
Where does `/tmp/README.md` come from? Can the asserted failure actually be caused by a File Not Found?

2 lines below it gets successfully uploaded, apparently (the CI says `ok 205 guest fileops`), but still a bit of a mystery 🙂

Reply:
The first arg `README.md` is the local [govc/test/README.md](https://github.com/vmware/govmomi/blob/master/govc/test/README.md).  We just need the container `/tmp` directory to exist.  We do need to handle more error cases (such as if /tmp didn't exist), but for now this just validates that auth is required.

See also: https://github.com/vmware/govmomi/blob/master/govc/USAGE.md#guestupload